run-tests.sh

```diff
@@ -0,0 +1,22 @@
+#!/bin/sh
+
+set -e
+
+build=no
+if [ "$1" = 'and' ] && [ "$2" = 'build' ]; then
```

Comment:
What is `and`?

Reply:
`$  ./run-tests and build`

Just being clever. I'll cut it down.

Reply:
That can be an option, `--build`.
I guess this script will be used in the CI, too?

Reply:
It doesn't need to be, it's just there for my own convenience.